modules/810-documentation/templates/service.yaml

```diff
@@ -26,7 +26,7 @@ spec:
   ports:
     - name: builder-http
       port: 8081
-      targetPort: 8081
+      targetPort: http-probe
```

Comment:
8081 is used for communicating with the builder container, in the first place. Should be something like http-builder, to make it clear.

Reply:
Fixed